src/main/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMessageHandler.java

```diff
@@ -52,21 +50,17 @@ public void handle(final InboundMeshMessage meshMessage) {
         logSentFor(interchange);
     }
 
-    private List<Transaction> getTransactions(Interchange interchange) {
-        return interchange.getMessages().stream()
-            .map(Message::getTransactions)
-            .flatMap(Collection::stream)
-            .collect(Collectors.toList());
+    private List<Message> getMessages(Interchange interchange) {
```

Comment:
This method is probably redundant now as it doesn't apply any additional logic

Reply:
Have removed this method and used `interchange.getMessages` instead in 697ebb8